tests/subsys/mgmt/mcumgr/smp_group_0/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+cmake_minimum_required(VERSION 3.20.0)
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
```

Comment:
We seem to do things differently in `tests/unit`: https://github.com/zephyrproject-rtos/zephyr/blob/main/tests/unit/crc/CMakeLists.txt

@nordic-krch @SebastianBoe @tejlmand

Reply:
Two types of testing are suppported. Integration testing and unit testing. When doing unit testing
the CMake should look like:

find_package(ZephyrUnittest REQUIRED HINTS $ENV{ZEPHYR_BASE})

But it's not clear to me what kind of testing we actually want to do here. Perhaps we want/need to do integration
testing due to the limitations/challenges of unit testing. See

https://docs.zephyrproject.org/latest/guides/test/ztest.html

Reply:
I'll settle with integration tests, seems more appropriate.